Address asciidoctor-dita-vale issues in Upgrading (#4588)
theforeman/foreman-documentation: guides/common/modules/proc_troubleshooting-permission-issues.adoc

[id="troubleshooting-permission-issues"]
= Troubleshooting permission issues

[role="_abstract"]
{Project} upgrades perform pre-upgrade checks.
If the pre-upgrade check discovers permission issues, you must identify and remedy the issues before proceeding with the upgrade.

> If the pre-upgrade check discovers permission issues, it fails with an error similar to the following one:

Reviewer: ```suggestion
A permission issue might look similar to the following one:
```

Just an idea because both line 8 and 10 start with "If the pre-upgrade check discovers permission issues,".
